src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -130,6 +130,7 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 			linkButtonContainer = DOM.$('li.action-item');
 			linkButtonContainer.setAttribute('role', 'presentation');
 			let linkButton = new Button(linkButtonContainer);
+			linkButton.title = this.buttonLink;
```

Comment:
Something like `Insert Link` or `Insert Image` seem clearer (see Github editor for an example)

Reply:
(some of the other buttons like the lists could use this too. Stuff like Highlight where it's an action rather than inserting something seems fine as is though)